iOS/Arirbnb/Arirbnb/DetailSearchScene/DetailSearchViewController.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  DetailSearchViewController.swift
+//  Arirbnb
+//
+//  Created by 지북 on 2021/05/18.
+//
+
+import UIKit
+
+class DetailSearchViewController: UIViewController{
+    static let storyboardName = "Main"
+    static let storybardID = "DetailSearchViewController"
+    static func create() -> DetailSearchViewController {
+        let storyboard = UIStoryboard(name: storyboardName, bundle: Bundle.main)
+        guard let vc = storyboard.instantiateViewController(identifier: storybardID) as? DetailSearchViewController else {
+            return DetailSearchViewController()
+        }
+        return vc
+    }
+    
+    enum Section: Int, CaseIterable {
+        case adjacentDestinations, searchResultDestinations
+        func sectionHeaderString() -> String? {
+            switch self {
+            case .adjacentDestinations: return "근처의 인기 여행지"
+            case .searchResultDestinations: return "검색된 장소"
+            }
+        }
+    }
+    
+    @IBOutlet weak var destinationsCollectionView: UICollectionView!
+    private var searchController: UISearchController!
+
+    private var destinations: [[Destination]] = [MockAdjacentDestination.mockDatas, MockSearchedDestinaion.mockDatas]
+    private var filteredDestinations: [Destination]!
```

Comment:
Array 타입에서 강제 옵셔널 보다 빈 배열을 초기값으로 할당 할 수 없을까요? 

```suggestion
    private var filteredDestinations: [Destination] = []
```